extensions/amp-story/0.1/amp-story.js

```diff
@@ -620,9 +620,9 @@ export class AmpStory extends AMP.BaseElement {
             page.setActive(false);
           });
         })
-        .then(() => this.switchTo_(firstPageEl.id))
+        .then(() => this.storeService_.dispatch(Action.PAGE_CHANGE,
```

Comment:
``switchTo_`` returns a promise, that needed to be resolved before the code continues to get executed. Are you sure it's not breaking anything?